cypress/e2e/webTables.cy.js

```diff
@@ -1,5 +1,127 @@
 describe('Web Tables page', () => {
-  it('', () => {
+
+  const user = {
+    firstName: 'John',
+    lastName: 'Doe',
```

Comment:
Email attribute should start with a lowercase 'e' (email instead of Email)

Reply:
Done